notebooks/2_read_write.ipynb

```diff
@@ -0,0 +1,684 @@
+{
```

Comment:
<p>functionalality -&gt; functionality</p><p> </p><p>There is also something wrong in the sentence " the reading functionalality provide the <code>read_csv</code> function to read the data into .."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>ok.</p>